src/Kernel/Context.class.st

```diff
@@ -537,7 +537,7 @@ Context >> completeCallee: aContext [
 		whileTrue: [
 			current := context.
 			nextContext := context quickStep.
-			nextContext ifNil: [ self halt ].
+			nextContext ifNil: [ self error: 'Should not happens!' ].
```

Comment:
should be "happen"

Reply:
Maybe a quick explanation of mhy it should not happen can help if one day it happen?

Reply:
Guys, I have no idea why. I just remove halt by error